src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,13 +24,34 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private DriveSystem driveSystem;
-  
+
+  private InstantCommand toggleFieldOriented; 
+  private InstantCommand toggleSlowMode;
+
   private DriveWithJoystick driveWithJoystick;
 
+  private Joystick joy;
+  private JoystickButton toggleFieldOrientedBtn;
+  private JoystickButton toggleSlowModeBtn;
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+
+    joy = new Joystick(0);
```

Comment:
Change to driver so that we can have driver and operator joysticks.

Reply:
*operator xbox controller, names are for clarity